src/Language/PureScript/Docs/RenderedCode/Types.hs

```diff
@@ -210,30 +185,6 @@ instance A.ToJSON RenderedCodeElement where
   toJSON (Symbol ns str link) =
     A.toJSON ["symbol", A.toJSON ns, A.toJSON str, A.toJSON link]
 
-asRenderedCodeElement :: Parse Text RenderedCodeElement
-asRenderedCodeElement =
-  tryParse "RenderedCodeElement" $
-    [ a Syntax "syntax"
-    , a Keyword "keyword"
-    , asSpace
-    , asSymbol
-    ] ++ backwardsCompat
-  where
-  a ctor' ctorStr = firstEq ctorStr (ctor' <$> nth 1 asText)
-  asSymbol = firstEq "symbol" (Symbol <$> nth 1 asNamespace <*> nth 2 asText <*> nth 3 asLink)
-  asSpace = firstEq "space" (pure Space)
-
-  -- These will make some mistakes e.g. treating data constructors as types,
-  -- because the old code did not save information which is necessary to
-  -- distinguish these cases. This is the best we can do.
-  backwardsCompat =
-    [ oldAsIdent
-    , oldAsCtor
-    ]
-
-  oldAsIdent = firstEq "ident" (Symbol ValueLevel <$> nth 1 asText <*> nth 2 (Link <$> asContainingModule))
-  oldAsCtor = firstEq "ctor" (Symbol TypeLevel <$> nth 1 asText <*> nth 2 (Link <$> asContainingModule))
-
```

Comment:
`asRenderedCodeElement` (along with `asRenderedCode`) is more substantial than most of the stuff I deleted. Pursuit doesn't use it (I compiled Pursuit locally against this branch to test), but is it possible something else does?

Reply:
This is probably an artifact of a previous failed attempt to have the Pursuit JSON blob contain `RenderedCode` rather than `Type`. If Pursuit still compiles against this branch I think it is fine to remove.